src/components/SidePanel/SidePanel.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import SwipeableDrawer from '@material-ui/core/SwipeableDrawer';
+import Button from '@material-ui/core/Button';
+import TextFields from './TextFields';
+import MenuIcon from '@material-ui/icons/Menu';
+
+
+
+const useStyles = makeStyles(theme => ({
+    list: {
+      width: 250,
+    },
+    fullList: {
+      width: 'auto',
+    },
+    sidePanel: {
+        width: 300,
+    },
+    iconButton: {
+        padding: 10,
+    },
+    button: {
+        margin: theme.spacing(1),
+      },
+    contentTitle: {
+        textAlign: 'left',
+    },
+    content: {
+        margin: theme.spacing(2),
+        textAlign: 'center',
+    },
+  }));
+  
+  export default function SidePanel() {
+    const classes = useStyles();
+    const [state, setState] = React.useState({
```

Comment:
`const [isOpen, setOpen] = React.useState` we should use just a variable like this, no need for checking the other sides